### DISCUSSION TITLE
Add set default value is $CURRENT_USER

### DISCUSSION THREAD

**Discussion by phongXEM** [directus/directus#23480] [Feature Requests]
### Summary

When M20 to the directus_user. If no value is passed, can set default value is the $CURRENT_USER

### Motivation

If this value is null, set default value as the current user

### Detailed Design

<img width="815" alt="Screenshot at Aug 25 23-15-11" src="https://github.com/user-attachments/assets/aa6a2d70-f3c8-4230-a833-c3a5c6d2267f">

### Requirements List

If this value is null, set default value as the current user

### Drawbacks

If this value is null, set default value as the current user

### Alternatives

If this value is null, set default value as the current user

### Adoption Strategy

If this value is null, set default value as the current user

**Comment by rijkvanzanten** [comment]
![CleanShot 2024-08-25 at 16 15 51@2x](https://github.com/user-attachments/assets/9d39ced0-eea4-404c-a930-3a1cf9601cd4)

You can do this today by setting "On Create" to "Save Current User ID" 👍🏻

**Reply by phongXEM** [reply]
"On Create" save current user id. but I want to pass user_id in, there is no need. is only accessible at the user's current.
